### PR TITLE
CHEAP-1 file bat added

### PR DESCRIPTION
**Pull request by mpaz-redstage**

### CODE REVIEW

File: aa234 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sf
+asf
```
Comment (mpaz-redstage) [inline aa234:2]: test!

**Review by mauriciopazpp — APPROVED** [review]
good!

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
change it!

**Review by mauriciopazpp — COMMENTED** [review]
test!

**Review by mauriciopazpp — COMMENTED** [review]
test!

**Review by mauriciopazpp — COMMENTED** [review]
Test!

**Review by mauriciopazpp — APPROVED** [review]
test!

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
Test!

**Review by mauriciopazpp — COMMENTED** [review]
Just a tip!

**Review by mauriciopazpp — APPROVED** [review]
Very good!

**Review by mauriciopazpp — CHANGES_REQUESTED** [review]
Change it!

**Review by mauriciopazpp — COMMENTED** [review]
test

**Pull request closed without merging**